### PR TITLE
Update Venetian translation 7.8.3 and fix

### PR DESCRIPTION
**Pull request by conky77**

### CODE REVIEW

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 1]
```diff
@@ -1183,42 +1200,47 @@ Vuto continuare?"/>
...
+            <word-chars-list-warning-end value=" ne ƚa to ƚista carateri."/>
             <cloud-invalid-warning value="Percorso no vaƚido."/>
-            <cloud-restart-warning value="Riavia Notepad++ per apƚicare ƚe modifiche."/>
+            <cloud-restart-warning value="Riavia Notepad++ par apƚicare ƚe modifiche."/>
             <cloud-select-folder value="Seƚesiona na carteƚa da dove Notepad++ posa ƚèxare o scrìvare ƚe proprie inpostasioni"/>
-            <shift-change-direction-tip value="Utiƚixa Shift+Invio per sercare ne ƚa diresion oposta"/>
+            <shift-change-direction-tip value="Utiƚixa Shift+Invio par sercare ne ƚa diresion oposta"/>
             <two-find-buttons-tip value="Modaƚità riserca a 2 botoni"/>
-            <find-status-top-reached value="Ruma: trovà ƚa prima ocorensa dal basso. Xe stà raggiunto el prinsipio de&apos;l documento."/>
+            <find-en-files-filter-tip value = "Ruma en cpp, cxx, h, hxx &amp;&amp; hpp:
```
Comment (xylographe) [inline PowerEditor/installer/nativeLang/venetian.xml]: Replace `<find-en-files-filter-tip` with `<find-in-files-filter-tip`.

File: PowerEditor/installer/nativeLang/venetian.xml [hunk 2]
```diff
@@ -1183,42 +1200,47 @@ Vuto continuare?"/>
...
+            <find-status-replace-end-reached value="Sostituisi: sostituìa ƚa prima ocorensa da ƚ&apos;alto. Se xe rivà a ƚa fine del documento "/>
+            <find-status-replace-top-reached value="Sostituìsi: sostituìa ƚa prima ocorensa dal baso. Xe stà ragiunto el prinsipio de el documento"/>
             <find-status-replaced-next-found value="Sostituisi: 1 ocorensa ƚa xe stà sostituìa. Trovà l&apos;ocorensa sucesiva"/>
             <find-status-replaced-next-not-found value="Sostituisi:1 ocorensa ƚa xe stà sostituìa. Nisuna ocorensa sucesiva"/>
             <find-status-replace-not-found value="Sostituisi: nisuna ocorensa trovà"/>
-            <find-status-replace-readonly value="Sostituisi: inposibiƚe sostituire el testo. El documento corente el xe in soƚa ƚetura"/>
+            <find-status-replace-readonly value="Sostituisi: inposibiƚe sostituire el testo. El documento corente el xe en soƚa ƚetura"/>
             <find-status-cannot-find value="Ruma: inposibiƚe trovare el testo &quot;$STR_REPLACE$&quot;"/>
-            <finder-find-in-finder value="Ruma in sti risultati..."/>
+            <finder-find-en-finder value="Ruma in sti risultati..."/>
```
Comment (xylographe) [inline PowerEditor/installer/nativeLang/venetian.xml]: Replace `<finder-find-en-finder` with `<finder-find-in-finder`.

**Comment by donho** [comment]
There are so many modification from v7.8.0 to v7.8.3, how come?

**Comment by conky77** [comment]
I mixed the modification for 7.8.3 with a better translation. I added "and fix" on the subject

**Comment by donho** [comment]
For the sake of making code review easy, your PR should contain only the modification:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests

Please fix the format/order as you want in this PR, So there won't be the same problem in the next PR. Let me know when it's OK.

**Comment by conky77** [comment]
Done. In the future I will change the order of the file only if the English order changes

**Pull request closed without merging**